src/MainPage/MainPage.css

```diff
@@ -0,0 +1,167 @@
+
```

Comment:
Been putting in the css work! Thank you. I'll check it out and play around with it, should be fun.